### ISSUE TITLE
A11y_AzureDataStudioSQLMigration_MigratetoAzureSQL_Databasebackup-networkshare_DeveloperTool:There is no label defined for the resource group combo box. 

### ISSUE THREAD

**Issue by fsteffi** [microsoft/azuredatastudio#22494]
[Check out Accessibility Insights!](https://accessibilityinsights.io/) - Identify accessibility bugs before check-in and make bug fixing faster and easier.”

GitHubTags: #A11yMAS;#A11yTCS;#A11ySev2;#SH_AZUREDATASTUDIOSQLMigration_June2021;#SQL Azure Data Studio;#Win10;#FTP;#DesktopApp;#WCAG4.1.2;#AINotInScope;#DevTool

Environment Details:
Application Name: SQL Azure Data Studio
Windows Version: Win10

Repro Steps:
Launch the application-Azure Data Studio-Insiders and install extension Azure SQL Migration.
Connect to any sql server.
Tab till Azure SQL migration Tab and press enter.
Tab till Migrate to azure sql and press enter.
Add azure account and tab till next button and press enter.
Select SQL server on azure virtual machine radio button and fill all the required fields and tab till next button and press enter.
Check Online migration radio button and tab till next button and press enter.
Check "My database are on a network share" radio button.
Tab till resource group combo-box.
Open Developer Tool using ctrl+shift+I and hover over to the combo box and check if it has name
Actual:
There is name defined for the resource group combo box as its label.
This issue is observed with the combo box throughout the application.

Expected:
There should be a proper name "resource group" defined for the combo box.

User Impact:
Screen reader users will not know the purpose of the combo box.

**Comment by fsteffi** [comment]
bug logged in ref to https://github.com/microsoft/azuredatastudio/issues/15764
